src/modules/dashboard/components/ActionsPage/ActionsPage.tsx

```diff
@@ -259,14 +266,35 @@ const ActionsPage = () => {
             />
           )}
           {!!events?.length && (
-            <ul>
+            <div style={{ width: '460px' }}>
```

Comment:
Wasn't this already set at 460? See here:
https://github.com/JoinColony/colonyDapp/blob/41f1217f1d93d82da32335078989d72bb412673e/src/modules/dashboard/components/ActionsPage/ActionsPage.css#L40-L43